python/lib/dependabot/python/update_checker/pipenv_version_resolver.rb

```diff
@@ -183,18 +165,19 @@ def handle_pipenv_errors(error)
             return if error.message.match?(/#{Regexp.quote(dependency.name)}/i)
           end
 
+          if error.message.match?(GIT_REFERENCE_NOT_FOUND_REGEX)
+            tag = error.message.match(GIT_REFERENCE_NOT_FOUND_REGEX).named_captures.fetch("tag")
+            # Unfortunately the error message doesn't include the package name.
+            # TODO: Talk with pipenv maintainers about exposing the package name, it used to be part of the error output
```

Comment:
tag @matteius in case you didn't see ☝️ ... not sure why this error no longer shows the end user the package name?